packages/platforms/delta/x86-64/x86-64-delta-agc5648s/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,238 @@
+/************************************************************
```

Comment:
@chenglin-tsai  This library appears to be in use by multiple platforms, each with its own copy. 
Can we we move this code to a common single library for all Delta platforms that need it?

Reply:
@jnealtowns Thanks for your suggestion.
Like @hans-tseng mentioned before (in delta_ag9032v1), "There are different numbers and types of CPLD in our platform and it might be difficult to share the plaform_lib for all delta platform"

Another reason is this platform(agc5648s) using BMC processor. Therefore, the functions in platform_lib.c are totally different than other Delta platforms. They called "IPMI user space commands" to monitor the hardware sensors.  And It might cause some problems to use it within a kernel module.